engine/crates/integration-tests/src/federation/builder/bench.rs

```diff
@@ -51,7 +51,7 @@ impl<'a> DeterministicEngineBuilder<'a> {
     }
 
     pub fn without_hot_cache(mut self) -> Self {
-        self.runtime.hot_cache_factory = InMemoryHotCacheFactory::inactive();
+        self.runtime.hot_cache_factory = InMemoryOperationCacheFactory::inactive();
```

Comment:
:bug: **Bug**
The change from `InMemoryHotCacheFactory` to `InMemoryOperationCacheFactory` in the `without_hot_cache` method might introduce a bug if `InMemoryOperationCacheFactory` does not have an `inactive` method or if its behavior differs from `InMemoryHotCacheFactory::inactive()`. This could lead to unexpected behavior when the `without_hot_cache` method is used, as the intended functionality might not be achieved.

```suggestion
        self.runtime.hot_cache_factory = InMemoryOperationCacheFactory::inactive(); // Ensure InMemoryOperationCacheFactory has an inactive method with the same behavior
```